src/app/tools/password-generator.component.html

```diff
@@ -10,15 +10,12 @@ <h1>{{'passwordGenerator' | i18n}}</h1>
     </div>
 </div>
 <div class="form-group">
-    <div class="form-check form-check-inline">
-        <input id="generate-password" name="type" value="password" class="form-check-input" type="radio"
-            (change)="saveOptions()" [(ngModel)]="options.type">
-        <label for="generate-password" class="form-check-label">{{'password' | i18n}}</label>
-    </div>
-    <div class="form-check form-check-inline">
-        <input id="generate-passphrase" name="type" value="passphrase" class="form-check-input" type="radio"
-            (change)="saveOptions()" [(ngModel)]="options.type">
-        <label for="generate-passphrase" class="form-check-label">{{'passphrase' | i18n}}</label>
+    <div class="form-check form-check-inline" appBoxRow name="PassTypeOptions" *ngFor="let o of passTypeOptions">
```

Comment:
* The `appBoxRow` directive is not used in web, please remove
* The `name` will be identical for both `<div>` elements generated by `*ngFor`. I don't think you need the `name` for anything, please remove

Reply:
@eliykat Removed the stuff, reverted jslib commit. Tested it out, works good.